src/styles.css

```diff
@@ -103,6 +110,7 @@ body {
   min-height: 40px;
   max-height: 450px;
   overflow-y: auto;
+  background-color: #fff;
```

Comment:
- [ ] [OPTIONAL] I think there should be a gap between the scores list and scores header containers. You can give top margins for the scores list container or you could use flex for the parent container and give gap value to it.